mslice/plotting/plot_window/slice_plot.py

```diff
@@ -199,14 +198,8 @@ def calc_figure_boundaries(self):
         bounds['x_label'] = fig_y * 0.05
         return bounds
 
-    def reset_info_checkboxes(self):
-        for key, line in six.iteritems(self._slice_plotter.overplot_lines[self.ws_name]):
-            if str(line.get_linestyle()) == 'None':
-                if isinstance(key, int):
-                    key = self._slice_plotter.get_recoil_label(key)
-                self.plot_window.disable_action(key)
-
     def toggle_overplot_line(self, key, recoil, checked, cif_file=None):
+        self.manager.report_as_current()
```

Comment:
I'm not sure if this is the best way to achieve this. I can see that something is required to ensure the overplot goes to the correct window but this will mean a change in behaviour with the keep/make current buttons.

@mducle Would this change to the current status make sense to users when switching intensities do you think? I can see it from both sides...

Reply:
The alternative is to pass the `Axes` object into the `slice_plotter` methods which I'm starting to think might be better.

Reply:
@MatthewBowles @martyngigg I think switching the current status back to the window where the user has changed the intensity is ok. I guess that it would be intuitive since if they clicked on that window to change the intensity, they should expect it to become current.

Reply:
I don't know what's causing that but it's fixed in #375.